ansible/roles/candlepin-user/tasks/test.yml

```diff
@@ -102,8 +102,8 @@
     dest: "artifacts/access.log"
     flat: yes
 
-- name: Evaluate result
-  command: "./parse-jtl.py -c {{item.value.result_file}} -b {{item.value.folder}}/{{item.value.baseline}} -e {{item.value.folder}}/{{item.value.expected}}"
+- name: Compare results
```

Comment:
I don't think it is necessary now. Both branches are passing with one `baseline.json` file. Add it please as task for somebody else for next sprint.